truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/source/Source.java

```diff
@@ -543,7 +544,16 @@ public void appendCode(CharSequence chars) {
      */
     public final SourceSection createSection(String identifier, int startLine, int startColumn, int charIndex, int length) {
         checkRange(charIndex, length);
-        return new SourceSection(null, this, identifier, startLine, startColumn, charIndex, length);
+        return createSectionImpl(identifier, startLine, startColumn, charIndex, length, SourceSection.EMTPY_TAGS);
+    }
+
+    public final SourceSection createSection(String identifier, int startLine, int startColumn, int charIndex, int length, String... tags) {
```

Comment:
Could we get Javadoc for this `createSection(..., tags)`? Would be great to know what these tags are supposed to be, and whether there are any standard tags (since the enum is gone it seems to be entirely up to the tools). Don't know whether tags are documented somewhere, but would be great to link there.